cmd/rukpakctl/cmd/unpack.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+
+*/
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// unpackCmd represents the unpack command
+var unpackCmd = &cobra.Command{
```

Comment:
I think we can either drop this unpack command, or apply the same feedback where we can use a constructor that returns a *cobra.Command, which the root.NewCmd constructor can register:

```go
package root

import (
    ...
)

func NewCmd() *cobra.Command {
    cmd := &cobra.Command{
        ...
    }

    cmd.AddCommand(unpack.NewCmd())
    ...
    return cmd
}
```